notesapi/v1/views.py

```diff
@@ -350,14 +350,13 @@ def post(self, *args, **kwargs):  # pylint: disable=unused-argument
 
     def delete(self, *args, **kwargs):  # pylint: disable=unused-argument
         """
-        Delete all annotations for user_id
-
+        Delete all annotations for user
```

Comment:
Too bad it's not called `user_id` - I'm left wondering whether `user` is a user ID, username, object, etc....

Reply:
I'd argue user_id was potentially worse because it's also not *the* user id since it's actually a hash of the user id.  I really should just clarify the docstring here.

Reply:
Agreed - that's worse!